cli/template/page-studs/_app/with-auth.tsx

```diff
@@ -1,8 +1,9 @@
 import "../styles/globals.css";
-import type { AppType } from "next/dist/shared/lib/utils";
+import type { AppType } from "next/app;
+import type { Session } from "next-auth";
 import { SessionProvider } from "next-auth/react";
 
-const MyApp: AppType = ({
+const MyApp: AppType<{ session: Session }> = ({
```

Comment:
Does this also not become `Session | null` like in [with-auth-trpc.tsx](https://github.com/t3-oss/create-t3-app/pull/485/files#diff-af435d5ed05b89c33e6bd87b6b094886fd9951b6677e4e1ef7d45ca6458269ef) ?

Reply:
Yeah missed that, updating